app/lib/submission_builder/ty2022/states/az/individual_return.rb

```diff
@@ -4,12 +4,15 @@ module Ty2022
     module States
       module Az
         class IndividualReturn < SubmissionBuilder::Document
-          FILING_STATUSES = {
-            single: 'Single',
-            married_filing_jointly: 'MarriedJoint',
-            married_filing_separately: 'MarriedFilingSeparateReturn',
-            head_of_household: 'HeadHousehold',
-          }.freeze
+          include DependentRelationshipTable
```

Comment:
I put the look up hash into a separate module so that this file wouldn't get so cluttered.